_pages/documentation/general_docs/m5ops.md

```diff
@@ -171,10 +173,18 @@ int main(void) {
 
 #ifdef GEM5
     m5_work_end_addr(0,0);
+    unmap_m5_mem();
 #endif
 }
 ```
 
+*Note*: You'll need to add a new header location for the compiler to find the `m5_mmap.h`.
+If you are following the example Makefile above, you can add the following line below where CFLAGS is defined,
+
+```c
+CFLAGS += $(GEM5_PATH)/util/m5/src/
```

Comment:
Shouldn't this be in `gem5/includes/gem5`?

Reply:
The include folder doesn't have m5_mmap.h

https://github.com/gem5/gem5/tree/6e39f2097d6f9a404668a0b89d6cc6b2108562b8/include/gem5

Reply:
Hmm.. Maybe it should, though. 

I don't like the idea of telling people to add the source path to their include list..
That said, I guess this is beyond the scope of this PR. I'll approve this PR, but could you create an issue for this?